test/deployments.ts

```diff
@@ -134,6 +134,77 @@ describe("Aludel factory deployments", function () {
           const program = await factory.programs(preexistingProgram.address);
           expect(program.name).to.eq("some name");
         });
+        describe("WHEN updates the program using update-program", () => {
+          let program: AludelFactory.ProgramDataStruct;
+          beforeEach(async () => {
+            program = await factory.programs(preexistingProgram.address);
+          });
+          describe("AND updates name", async () => {
+            beforeEach(async () => {
+              await run("update-program", {
+                program: preexistingProgram.address,
+                newName: "a brave new name",
+              });
+            });
+            it("THEN only the name is changed", async () => {
+              const updatedProgram = await factory.programs(
+                preexistingProgram.address
+              );
+              expect(updatedProgram.name).to.eq("a brave new name");
+              expect(updatedProgram.stakingTokenUrl).to.eq(
+                program.stakingTokenUrl
+              );
+              expect(updatedProgram.template).to.eq(program.template);
+              expect(updatedProgram.startTime).to.eq(program.startTime);
+            });
+          });
+          describe("AND updates url", async () => {
```

Comment:
fix the grammar of the descriptions.

this should perhaps be `AND WHEN updating only the url`

The test logs are harder to follow otherwise

Reply:
wouldn't `WHEN` be redundant?
```
          WHEN updates the program using update-program
            AND updates the name
              ✓ THEN only the name is changed
```